dvc/ignore.py

```diff
@@ -52,16 +52,12 @@ def __init__(self, pattern_list, dirname):
         ]
 
     @classmethod
-    def from_files(cls, ignore_file_path, fs, root):
-        assert os.path.isabs(ignore_file_path)
-        dirname = os.path.normpath(os.path.dirname(ignore_file_path))
-        ignore_file_rel_path = os.path.relpath(ignore_file_path, root)
-        with fs.open(ignore_file_path, encoding="utf-8") as fobj:
+    def from_file(cls, path, fs, name):
```

Comment:
The name was misleading as it only expected one file.

Reply:
Also `ignore_file_path` is unnecessarily long, as in this context it is very clear that we are talking about the dvcignore file path.